### PR TITLE
Devex pdf order

### PR DESCRIPTION
**Pull request by matthopson**
Generate order pdf via react template.

Screen Shot 2020-05-27 at 9 55 15 AM

### CODE REVIEW

**Comment by rachaelparris** [comment]
I think the document title is supposed to be all caps, and not underlined, unless UX decided to change this?

File: shared/src/business/useCases/courtIssuedOrder/createCourtIssuedOrderPdfFromHtmlInteractor.js [hunk 1]
```diff
@@ -1,78 +1,60 @@
...
+      docketNumberWithSuffix,
+      orderContent: contentHtml,
+      orderTitle: documentTitle,
+      signatureText,
+    },
   });
+
+  return await applicationContext
+    .getUseCaseHelpers()
+    .saveFileAndGenerateUrl({ applicationContext, file: orderPdf });
```
Comment (rachaelparris) [inline shared/src/business/useCases/courtIssuedOrder/createCourtIssuedOrderPdfFromHtmlInteractor.js]: Should this go to the temp bucket?

**Comment by matthopson** [comment]
It's supposed to be underlined. I probably just didn't cap it in the test data that generates it.

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/useCases/courtIssuedOrder/createCourtIssuedOrderPdfFromHtmlInteractor.js (hunk 1 above)
Comment (matthopson) [reply shared/src/business/useCases/courtIssuedOrder/createCourtIssuedOrderPdfFromHtmlInteractor.js]: I don't _think_ so? But I'm wondering if we need to generate previews differently now.

**Pull request merged**